### PR TITLE
MEGALimited.MEGAcmd version 1.5.0.0

### PR DESCRIPTION
**Pull request by MatyiFKBT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55683)

### CODE REVIEW

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
+PackageVersion: 1.5.0.0
+PackageLocale: en-US
+Publisher: MEGA Limited
+PackageName: MEGAcmd
+License: Freeware
+ShortDescription: MEGAcmd
```
Comment (ItzLevvie) [inline manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml:10]:
```suggestion
ShortDescription: MEGAcmd provides non UI access to MEGA services. It intends to offer all the functionality with your MEGA account via commands.
```

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
```
Comment (ItzLevvie) [inline manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MEGA.MEGAcmd
```

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
```
Comment (ItzLevvie) [inline manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.installer.yaml:4]:
```suggestion
PackageIdentifier: MEGA.MEGAcmd
```

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
```
Comment (ItzLevvie) [inline manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.yaml:4]:
```suggestion
PackageIdentifier: MEGA.MEGAcmd
```

Comment (ItzLevvie) [reply manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.yaml:4]: You will also need to update the folder from `MEGALimited` to `MEGA`.

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml:4]: You will also need to update the folder from `MEGALimited` to `MEGA`.

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: MEGALimited.MEGAcmd
+PackageVersion: 1.5.0.0
+PackageLocale: en-US
+Publisher: MEGA Limited
```
Comment (ItzLevvie) [inline manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.locale.en-US.yaml:7]:
```suggestion
Publisher: Mega Limited
```
This must match Control Panel.

File: manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.installer.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/m/MEGALimited/MEGAcmd/1.5.0.0/MEGALimited.MEGAcmd.installer.yaml:4]: You will also need to update the folder from `MEGALimited` to `MEGA`.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by MatyiFKBT** [comment]
@ItzLevvie is validation failing because I missed one of the edits?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> @ItzLevvie is validation failing because I missed one of the edits?

No.

It's flagged by anti-virus software.

**Pull request closed without merging**